From 5d0d3961591314ba862c6dd51fd915f08b3c81d0 Mon Sep 17 00:00:00 2001
From: Dorian Stoll <dorian.stoll@tmsp.io>
Date: Sun, 23 Jul 2023 17:55:15 +0200
Subject: [PATCH 35/62] Update IPTS from module repo

Changes:
 * Fix swapped parameters in EDS v2

Based on https://github.com/linux-surface/intel-precise-touch/commit/4b5c013d0140c47e4a009adf9a053a45bb0532a3

Signed-off-by: Dorian Stoll <dorian.stoll@tmsp.io>
Patchset: ipts
---
 drivers/hid/ipts/eds2.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/hid/ipts/eds2.c b/drivers/hid/ipts/eds2.c
index 7c5cbafdb279..198dc65d7887 100644
--- a/drivers/hid/ipts/eds2.c
+++ b/drivers/hid/ipts/eds2.c
@@ -138,7 +138,7 @@ int ipts_eds2_raw_request(struct ipts_context *ipts, u8 *buffer, size_t size, u8
 		return -EIO;
 
 	if (request_type == HID_REQ_GET_REPORT)
-		return ipts_eds2_get_feature(ipts, buffer, report_id, size, feedback_type);
+		return ipts_eds2_get_feature(ipts, buffer, size, report_id, feedback_type);
 	else
-		return ipts_eds2_set_feature(ipts, buffer, report_id, size, feedback_type);
+		return ipts_eds2_set_feature(ipts, buffer, size, report_id, feedback_type);
 }
-- 
2.39.1

